### ISSUE TITLE
website accessibility tests fail

### ISSUE THREAD

**Issue by fniedtner** [microsoft/autogen#5579]
### What happened?

https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanConsolidatedReport?siteId=9fbd8719-0d90-4752-3097-08dbfa93480f

### What did you expect to happen?

tests pass

### How can we reproduce it (as minimally and precisely as possible)?

check link

### AutoGen version

0.4

### Which package was this bug in

Other

**Comment by Vidit-Ostwal** [comment]
@fniedtner, showing you don't have access to this page

**Comment by jackgerrits** [comment]
#5629
